2.7.0/done/result_distributed (checkpoint).md

```diff
@@ -29,21 +29,24 @@ The categories below are as follows:
 ### bc breaking
 ### deprecation
 ### new features
+- [DCP] Cache save plans: planner helpers and interface updates ([#147116](https://github.com/pytorch/pytorch/pull/147116))
+- [DCP] Cache save plans in default planner ([#147343](https://github.com/pytorch/pytorch/pull/147343))
```

Comment:
Can you combine these into one bullet as they're related? And then remove the [dcp] tag in the front